notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>Please follow the standardized steps to set up project ID and the environment. See notebook_template https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/36455b8125802459f3a40752fbda0e4c9407c854/notebooks/notebook_template.ipynb </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>